### PR TITLE
Prometheus to scrap CPU and RAM usage

### PR DESCRIPTION
**Pull request by Archana-GS**
Adding Prometheus to scrap metrics i.e cpu and ram usage

### CODE REVIEW

File: ai_ml_app.py [hunk 1]
```diff
@@ -5,8 +5,16 @@
 """
 from flask import Flask, jsonify, render_template, request
 import is_pto.is_pto as pto_classifier
+from flask import Flask, Response
```
Comment (qxf2) [inline ai_ml_app.py:8]: Why is Flask being imported again here? It is already imported in line 6. Move Response to line 6.

File: ai_ml_app.py [hunk 2]
```diff
@@ -25,9 +33,19 @@ def is_pto():
...
+    cpu_usage.set(cpu_percent)
+    ram_usage.set(ram_info.percent)
+
+    # generate Prometheus metrics in the appropriate format
+    return Response(generate_latest(), mimetype="text/plain")
+
 #---START OF SCRIPT
 if __name__ == '__main__':
-    app.run(host='0.0.0.0', port=6464)
+    app.run(debug=True , host='0.0.0.0', port=6464)
```
Comment (qxf2) [inline ai_ml_app.py:51]: Why has debug been turned on? You should not use it on production.

File: tests/requirements.txt [hunk 3]
```diff
@@ -3,5 +3,3 @@ requests==2.25.1
 fastdiff==0.2.0 
 snapshottest==0.6.0
 termcolor==1.1.0
-
-
```
Comment (qxf2) [inline tests/requirements.txt:7]: Why did this file change at all?

**Review by qxf2 — CHANGES_REQUESTED** [review]
Please address the review comments.

File: tests/requirements.txt (hunk 3 above)
Comment (Archana-GS) [reply tests/requirements.txt:7]: just removed extra lines that it

File: ai_ml_app.py (hunk 2 above)
Comment (Archana-GS) [reply ai_ml_app.py]: ok , removed debug mode

File: ai_ml_app.py (hunk 1 above)
Comment (Archana-GS) [reply ai_ml_app.py]: Moved 'Response' to 6th line

**Pull request merged**